addons/l10n_ec/views/root_sri_menu.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem
+        id="sri_menu"
+        name="Ecuadorian SRI"
+        parent="account.menu_finance_configuration"
+        sequence="25"/>
```

Comment:
we definitively need to find a way to remove the custom payment methods from l10n and use only the standard ones. I guess for now, we have no choice to keep them as it is